front/src/utils/strings.ts

```diff
@@ -34,3 +36,11 @@ export function snakeToCamel(str: string) {
 }
 
 export const emptyStringRegex = /^\s+$/;
+
+export function joinTranslationList(translationList: string, item: string): string {
```

Comment:
Maybe it's better, I'm not sure
```suggestion
export function getTranslationKey(translationList: string, item: string): string {
```